source/administration/config-settings.rst

```diff
@@ -727,6 +727,35 @@ Test Connection
 
 Ensures that the user can access the server and that the settings are valid.
 
+Enable Document Search by Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File search is available now in Mattermost Cloud, and in Mattermost Self-Managed from v5.35, with mobile support coming soon.
+
+**True**: When **true**, supported document types are searchable by their content. Install `these dependencies <https://github.com/sajari/docconv#dependencies>`__ to extend content searching support to include DOCX, RTF, and PAGES files. 
+
+**False**: Supported document types aren't searchable by their content. When document content search is disabled, users can search for files by file name only.
+
++---------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"XXX": true`` with options ``true`` and ``false``.                      |
++---------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+  Search results for documents shared before the file search was available may be incomplete until a `data migration is executed using the CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-extract-documents-content>`__. 
+
+Enable Searching Content of Documents within ZIP Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+File search is available now in Mattermost Cloud, and in Mattermost Self-Managed from v5.35, with mobile support coming soon.
+
+**True**: When **true**, content of documents within ZIP files are returned in search results. This may have an impact on server performance for large files. 
+
+**False**: The content of documents within ZIP files aren't returned in search results.
+
++---------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"XXX": false`` with options ``true`` and ``false``.                     |
```

Comment:
@jespino - What is the name of this configuration setting in the config.json file?